### DISCUSSION TITLE
Missing digital signature in windows CLI installer

### DISCUSSION THREAD

**Discussion by MyOwnByte** [forcedotcom/cli#2640] [General]
Is there a special reason why Windows installer is packaged without Digital signature? 
Last installer with digital signature was 2.21.5
"2.21.5": "https://developer.salesforce.com/media/salesforce-cli/sf/versions/2.21.5/6181927/sf-v2.21.5-6181927-x64.exe",
2.21.6 and following installers are not signed:
"2.21.6": "https://developer.salesforce.com/media/salesforce-cli/sf/versions/2.21.6/8a2e193/sf-v2.21.6-8a2e193-x64.exe",
Not sure if this is a bug or not, but doesn't look safe for end-users.
![windefender](https://github.com/forcedotcom/cli/assets/87585535/4e82e252-d81e-4ca6-83cb-1ffa5e33e08d)

**Comment by mshanemc** [comment]
We found the problem in the signing pipeline and they should all be signed now.  thanks for letting us know!
